index.html

```diff
@@ -38,26 +46,33 @@ <h2>I am a Software Developer</h2>
   </section>
 
   <section class="works-section">
-    <h2>My Recent Works</h2>
-    <hr>
+    <div id="works-header">
+      <h2>My Recent Works</h2>
+      <hr>
+    </div>
     <div class="cards-container">
       <div class="card">
```

Comment:
- [ ] Please make sure that when the project is hovered on with the mouse the description and dark opacity disappears and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository. Currently, button that appears on the hover is missing. This is how the card should look when hovered: 
<img width="694" alt="Screenshot 2022-03-31 at 19 24 03" src="https://user-images.githubusercontent.com/77038610/161091566-335f6b08-a7b5-4769-b448-008a0675b733.png">

Reply:
Hi @ShoiraTa 

Thanks for your kind and detailed review.

I have endeavoured to fix them up. However, it seems solving this current conversation will require making use of JavaScript. The project seems not to require implementing JavaScript functionalities.

Reply:
Hi @emmiiorji,sorry for a late response. I guess you already know by now that these changes can be done with the help of CSS

Reply:
Yes, @ShoiraTa. It was a good spur to learn about that.

Thank you once again.